package.json

```diff
@@ -1,25 +1,29 @@
 {
-  "name": "zoom-talking-stick",
+  "name": "zoom-shuffle-bot",
   "version": "1.0.0",
   "description": "",
   "main": "chatbot.js",
   "type": "module",
   "scripts": {
     "test": "npm run test:unit && npm run test:integration",
     "test:unit": "jest --testPathIgnorePatterns=test/integration",
-    "test:integration": "jest --testPathPattern=test/integration --runInBand",
+    "test:integration": "npm run db:in-tests:up && sleep 1 && jest --testPathPattern=test/integration --runInBand && npm run db:in-tests:down",
```

Comment:
having a static sleep is a perfect recipe for brittle tests. let's use something which waits until the server accepts connections. I'm sure there are established ways to do this